lua/lspconfig/cssls.lua

```diff
@@ -21,15 +21,17 @@ configs[server_name] = {
   };
   docs = {
     description = [[
-https://github.com/vscode-langservers/vscode-css-languageserver-bin
+
+    
+https://github.com/hrsh7th/vscode-langservers-extracted
 
 `css-languageserver` can be installed via `npm`:
+
 ```sh
-npm install -g vscode-css-languageserver-bin
+npm i -g vscode-langservers-extracted
 ```
-
-Neovim does not currently include built-in snippets. `vscode-css-languageserver` only provides completions when snippet support is enabled.
-To enable completion, install a snippet plugin and add the following override to your language client capabilities during setup.
+    
+Neovim does not currently include built-in snippets. vscode-css-language-server only provides completions when snippet support is enabled. To enable completion, install a snippet plugin and add the following override to your language client capabilities during setup.
```

Comment:
Out of curiosity, why did you remove the backticks around the server name?

Reply:
Sheer stupidity!